llvm/test/tools/UpdateTestChecks/update_mc_test_checks/Inputs/amdgpu_asm_err.s.expected

```diff
@@ -0,0 +1,5 @@
+; NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py
+// RUN: not llvm-mc -triple=amdgcn -show-encoding %s 2>&1 | FileCheck --check-prefixes=CHECK %s
+
+v_bfrev_b32 v5, v299
+// CHECK: :[[@LINE-1]]:{{[0-9]+}}: error: register index is out of range
```

Comment:
Can we match the actual column from the error message?

Reply:
Hi Alex are you talking about the actual line number here? I made it this way since I found most of the test are using this regex to match line number.

Reply:
No I meant the column number, i.e. the second component - using `[[#@LINE-1]]` is good since it means no changes if new lines are added.

Reply:
oh, this might requires more changes. The current approach here is to extract and test line-by-line, so the line number extracted from this python script is not the actual line number when running in llvm-lit

Reply:
Added